### PR TITLE
[Edit] JavaScript Add Codebytes Issue: #2407

### PR DESCRIPTION
**Pull request by Jamesllllllllll**
### Description

Closes [#2407](https://github.com/Codecademy/docs/issues/2407)

Files edited:

1. [javascript/concepts/promise/terms/all](https://github.com/Codecademy/docs/blob/main/content/javascript/concepts/promise/terms/all/all.md)
2. [javascript/concepts/promise/terms/any](https://github.com/Codecademy/docs/blob/main/content/javascript/concepts/promise/terms/any/any.md)

Added Codebytes and corrected typos.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
  - Fixed one typo in the example output in promise.all()
  - Fixed one typo in the example code in promise.any() (extra closing parenthesis)

- Updating the documentation
  - Added Codebytes to both promise.all() and promise.any()

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/promise/terms/all/all.md [hunk 1]
```diff
@@ -56,6 +56,43 @@ Promise.all([promiseA, promiseB])
 The output would be:
 
 ```plaintext
-Results from Promise.all(): [144,42]
+Results from Promise.all(): [23,144]
 Operations for Promise.all() have finished.
 ```
+
+## Codebyte Example
+
```
Comment (SSwiniarski) [inline content/javascript/concepts/promise/terms/all/all.md:64]: We need some introductory sentence for the codebyte that gives a brief overview of what it does.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Jamesllllllllll I did a first review and left one comment.

File: content/javascript/concepts/promise/terms/all/all.md (hunk 1 above)
Comment (Jamesllllllllll) [reply content/javascript/concepts/promise/terms/all/all.md:64]: Thank you! I added an introductory sentence. I also clicked "Update branch" below, I'm not sure if I was supposed to do that :)

This my first time contributing to any open source project :D

**Review by SSwiniarski — APPROVED** [review]
@Jamesllllllllll Looks good for a second review.

File: content/javascript/concepts/promise/terms/all/all.md [hunk 2]
```diff
@@ -56,6 +56,45 @@ Promise.all([promiseA, promiseB])
 The output would be:
 
 ```plaintext
-Results from Promise.all(): [144,42]
+Results from Promise.all(): [23,144]
 Operations for Promise.all() have finished.
 ```
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/javascript/concepts/promise/terms/all/all.md:63]: Looks like the .all() method was erroneously included in multiple issues. This one should be reverted.

File: content/javascript/concepts/promise/terms/any/any.md [hunk 3]
```diff
@@ -65,3 +65,32 @@ Promise.any(promises)
   })
   .finally(() => console.log('Operations on Promise.any() have finished.'));
 ```
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/javascript/concepts/promise/terms/any/any.md:69]: This codebyte is essentially the same as example 2 in the existing entry. We do not include duplicates in our entries, and ideally each example demonstrates some new functionality about the method, the implementation, special cases, etc. Please revise.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Jamesllllllllll I've added a couple of comments. Please update, and tag me when you've finished making changes. Thanks

File: content/javascript/concepts/promise/terms/any/any.md (hunk 3 above)
Comment (Jamesllllllllll) [reply content/javascript/concepts/promise/terms/any/any.md:69]: Thank you for your feedback! I will change the example to demonstrate how Promise.all() is resolved when the first resolved promise is found.

**Review by Jamesllllllllll — COMMENTED** [review]
Revised Codebyte to demonstrate resolving after finding a rejected Promise, and when the first resolved Promise is found.

File: content/javascript/concepts/promise/terms/all/all.md (hunk 2 above)
Comment (Jamesllllllllll) [reply content/javascript/concepts/promise/terms/all/all.md]: I reverted my commits on the .all() method @caupolicandiaz

**Comment by caupolicandiaz** [comment]
@Jamesllllllllll Unfortunately, the `any` function isn't currently supported by our compiler, so, the codebyte results in an error. I'm going to close this PR and issue. I'm sorry this entry didn't work out, I would recommend that you look for another issue that you'd like to work on, and we can help you move forward with a new contribution. Thanks

**Comment by caupolicandiaz** [comment]
Issue was a partial duplicate and the remaining function does not run within a codebyte.

**Comment by Jamesllllllllll** [comment]
Ok, thank you! @caupolicandiaz

**Pull request closed without merging**